catalog/imaging/imaging-actionprovider-chip/src/test/java/org/codice/alliance/imaging/chip/actionprovider/TestImageChipActionProvider.java

```diff
@@ -64,8 +71,8 @@ public void testCanHandleInvalidMetacard() {
 
     @Test
     public void testCanHandleInvalidLocationOnMetacard() {
```

Comment:
Bug discovered! Updated code to perform more robust validation checking on location format, beyond simple non-null check. Added unit test for non-null, invalid WKT location.